parm/FV3LAM_wflow.xml

```diff
@@ -753,6 +760,49 @@ the <task> tag to be identical to the ones above for other output times.
   {%- endif %}
 {%- endif %}
 
+{%- if run_task_run_prdgen %}
+<!--
+************************************************************************
+************************************************************************
+-->
+<metatask name="&RUN_PRDGEN_TN;{{ uscore_ensmem_name }}">
+
+    <var name="fhr">{% for h in range(0, fcst_len_hrs+1) %}{{ " %03d" % h }}{% endfor %}</var>
+    <task name="&RUN_PRDGEN_TN;{{ uscore_ensmem_name }}_f#fhr#" cycledefs="forecast" maxtries="{{ maxtries_run_prdgen }}">
+
+      &RSRV_PRDGEN;
+      <command>&LOAD_MODULES_RUN_TASK_FP; "&RUN_PRDGEN_TN;" "&JOBSdir;/JREGIONAL_RUN_PRDGEN"</command>
+
+      <nodes>{{ nnodes_run_prdgen }}:ppn={{ ppn_run_prdgen }}</nodes>
+      <walltime>{{ wtime_run_prdgen }}</walltime>
+      <nodesize>&NCORES_PER_NODE;</nodesize>
+      <native>&SCHED_NATIVE_CMD;</native>
+
+      <jobname>&RUN_PRDGEN_TN;{{ uscore_ensmem_name }}_f#fhr#</jobname>
+      <join>&LOGDIR;/&RUN_PRDGEN_TN;{{ uscore_ensmem_name }}_f#fhr#<cyclestr>_@Y@m@d@H</cyclestr>&LOGEXT;</join>
+
+      <envar><name>GLOBAL_VAR_DEFNS_FP</name><value>&GLOBAL_VAR_DEFNS_FP;</value></envar>
+      <envar><name>USHdir</name><value>&USHdir;</value></envar>
+      <envar><name>PDY</name><value><cyclestr>@Y@m@d</cyclestr></value></envar>
+      <envar><name>cyc</name><value><cyclestr>@H</cyclestr></value></envar>
+      <envar><name>subcyc</name><value><cyclestr>@M</cyclestr></value></envar>
+      <envar><name>LOGDIR</name><value>&LOGDIR;</value></envar>
+      <envar><name>SLASH_ENSMEM_SUBDIR</name><value><cyclestr>{{ slash_ensmem_subdir }}</cyclestr></value></envar>
+      <envar><name>ENSMEM_INDX</name><value><cyclestr>#{{ ensmem_indx_name }}#</cyclestr></value></envar>
+
+      <envar><name>CYCLE_TYPE</name><value><cyclestr>prod</cyclestr></value></envar>
```

Comment:
Would this be set differently, or is it always `CYCLE_TYPE=prod`? If it is more or less always the same, does it make sense to move it to be hard-coded in the ex-script? Is it an NCO requirement to be exported?

I suppose the same questions hold for `CYCLE_SUBTYPE` and `TMMARK` below.

Reply:
These variables can be revisited in a later PR, I'll remove them for this one.